mmv1/products/healthcare/api.yaml

```diff
@@ -379,6 +379,29 @@ objects:
                         resource is a recursive structure; when the depth is 2, the CodeSystem table will have a column called
                         concept.concept but not concept.concept.concept. If not specified or set to 0, the server will use the default
                         value 2. The maximum depth allowed is 5.
+      - !ruby/object:Api::Type::Array
+        name: notificationConfigs
```

Comment:
As an array type, I believe this needs a description to build

Reply:
Hi Cameron,

Thanks for pointing this out. I am new here and I am still learning how to run tests, even how to use git properly. I'll come back to this PR once I figure out the proper workflow.

Reply:
No worries, you've done well already! There's a lot of context to be learned. If you haven't seen it already, check out https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests for our testing guidelines

Reply:
No, after this PR is merged you will not have to do any additional steps